packages/sdk/src/node.ts

```diff
@@ -0,0 +1,117 @@
+function importNodeModule<T>(name: string): Promise<T> {
```

Comment:
@2heal1  @zhoushaw how about this? it lets me avoid having to pass require / non_webpack_require, and keeps it agnostic of webpack couplings. 
Will this work better than trying to pass requireFunc down to SDK?